vnext/ReactWindowsCore/WinRTWebSocketResource.h

```diff
@@ -0,0 +1,128 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include <IWebSocketResource.h>
+#include <dispatchQueue/dispatchQueue.h>
+#include <eventWaitHandle/eventWaitHandle.h>
+#include <winrt/Windows.Networking.Sockets.h>
+#include <winrt/Windows.Storage.Streams.h>
+
+// PPL
+#include <concurrent_queue.h>
+
+// Standard Library
+#include <future>
+#include <queue>
+
+namespace Microsoft::React {
+
+class WinRTWebSocketResource : public IWebSocketResource, public std::enable_shared_from_this<WinRTWebSocketResource> {
+  winrt::Windows::Foundation::Uri m_uri;
```

Comment:
I think Nikolai mentioned this before, but the generally prefered order for class definitions is:
1. Public methods
2. Private methods
3. Private variables

Reply:
Will re-arrange in next PR.

Reply:
Seems like this could be addressed pretty easily. Why delay?